DashAI/back/plugins_system/utils.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+if sys.version_info < (3, 10):
+    from importlib_metadata import entry_points
+else:
+    from importlib.metadata import entry_points
+
+
+def update_plugins(registry_dict_plugins):
+    # Retrieve plugins groups (DashAI components)
+    task_plugins = entry_points(group="dashai.plugins.task")
+    model_plugins = entry_points(group="dashai.plugins.model")
+    plugin_groups = {"task": task_plugins, "model": model_plugins}
+
+    for component, plugins in plugin_groups.items():
```

Comment:
```suggestion
    for component, plugins in zip(["task", "model"], [task_plugins, model_plugins]):
```
There is no need to define a plugin_groups dict if you are only iterating on it